amqp/src/main/java/io/github/tcdl/msb/adapters/amqp/AmqpConsumerAdapter.java

```diff
@@ -20,10 +21,10 @@
     private String consumerTag;
     private AmqpBrokerConfig adapterConfig;
     private boolean isResponseTopic = false;
+    private Optional<String> currentQueueName = Optional.empty();
```

Comment:
I would not use Optional for fields (as a return value Optional is fine).

```
Optional is primarily intended for use as a method return type where there is a clear need to represent "no result," and where using null is likely to cause errors.
```

Reply:
What would you like to see instead? I can't think of a better way to express a field value that may or may not be present depending on the state of the instance.

Reply:
I would just use String for field type. In this case absence of value is represented by null.